WordPress/build.gradle

```diff
@@ -64,6 +64,13 @@ android {
         buildConfigField "boolean", "NEW_SITE_CREATION_ENABLED", "true"
         buildConfigField "boolean", "OFFER_GUTENBERG", "true"
         buildConfigField "boolean", "DOMAIN_REGISTRATION_ENABLED", "false"
+
+        // Gutenberg's dependency - react-native-video is using
+        // Java API 1.8
+        compileOptions {
+            sourceCompatibility JavaVersion.VERSION_1_8
+            targetCompatibility JavaVersion.VERSION_1_8
+        }
```

Comment:
wondering, given this is a React Native requisite, is there a way we can move this gradle block to the RN bridge instead of adding this to the WordPress app? Not sure if it's possible though, but sounds like it belongs there rather than here

Reply:
Hey @mzorz, thank for the comment.
I tried to put this into the module, but it didn't pass. Let me take one more chance with it, maybe something can be done. I also don't like this, so let try to move it :)

Reply:
> Let me take one more chance with it, maybe something can be done. I also don't like this, so let try to move it :)

👋 @marecar3 , curious what was the result of the extra effort on this one. I'm assuming that since this PR got merged with this change in, the effort didn't lead to something usable but still, can you share your findings if any? Thanks!

Reply:
Thanks, @hypest for returning on this one. I tried to move those into `react-native-gutenberg-bridge` but without luck. Also, I have done some googling and tried some of my own ideas but not sure how this can be resolved. If you have some ideas, please share with me and I am willing to try them :)

Reply:
Sharing this for context: https://github.com/react-native-community/react-native-video#exoplayer-290-java-18-requirement

Reply:
Maybe this needs to be moved to `react-native-video` rather than the bride itself? I agree its not ideal to have this here, but if there's no alternative then we can live with it.

Reply:
Hey @jtreanor, `react-native-video` already has set those options https://github.com/wordpress-mobile/react-native-video/blob/enable-android-build-of-standalone-package/android-exoplayer/build.gradle
but for some reason, the main project (WPAndroid) isn't using them in a process of the build.

Not sure how we can force Main project to use those options @hypest @jtreanor